composer.json

```diff
@@ -11,23 +11,24 @@
     "type": "library",
     "license": "BSD-3",
     "homepage": "https://www.phalconphp.com",
-    "authors" : [
+    "authors": [
         {
-            "name" : "Andres Gutierrez",
-            "email" : "andres@phalconphp.com"
+            "name": "Andres Gutierrez",
+            "email": "andres@phalconphp.com"
         },
         {
-            "name" : "Mark Garrett",
-            "email" : "mark@moderndeveloperllc.com"
+            "name": "Mark Garrett",
+            "email": "mark@moderndeveloperllc.com"
         }
     ],
     "require": {
-        "php": ">=5.5",
-        "ext-phalcon": "~3.0",
+        "php": ">=7.2",
+        "ext-phalcon": "4.0.0-beta.2",
```

Comment:
I'm not sure if we can track version with ext-phalcon. I think we need to use * here. cc @sergeyklay 
Maybe add ext-psr as well?